infra/gcp/bash/lib.sh

```diff
@@ -80,10 +80,10 @@ readonly AUDITOR_INVOKER_SVCACCT="k8s-infra-gcr-auditor-invoker"
 readonly AUDITOR_SERVICE_NAME="cip-auditor"
 
 # The service account name for the file promoter.
-readonly FILE_PROMOTER_SVCACCT="k8s-infra-promoter"
+export FILE_PROMOTER_SVCACCT="k8s-infra-promoter"
```

Comment:
Why this is needed ?

Reply:
These variables are now used outside the script, in `ensure-releng.sh`

Reply:
Mmh I thought it was needed to access the vars outside the lib. Dropped the commit.